backoffice/src/views/DeleteBeaconView.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Radio,
+  RadioGroup,
+} from "@mui/material";
+import { Field, Form, FormikErrors, FormikProps, withFormik } from "formik";
+import { FunctionComponent } from "react";
+import { PageHeader } from "../components/layout/PageHeader";
+import { PageContent } from "components/layout/PageContent";
+import { IBeaconsGateway } from "gateways/beacons/IBeaconsGateway";
+import { IDeleteBeaconDto } from "entities/IDeleteBeaconDto";
+import { DeleteBeaconFormValues } from "lib/DeleteBeaconFormValues";
+import { BeaconDeletionReasons } from "lib/BeaconDeletionReasons";
+import { useHistory } from "react-router-dom";
+
+interface IDeleteBeaconViewProps {
+  beaconsGateway: IBeaconsGateway;
```

Comment:
We have two DeleteBeaconViews: one for legacy and one for modern. I would like to have just the one reused between both beacon types. Not sure how to tell the given view to call the correct gateway method for its beacon type

Reply:
I guess you don't need to. If they have the same reasons you can work out if it's legacy or modern in the api.

If there's logic that depends on it, could always pass though a bool into the props